patch: add support dp_memory for tang primer 20k board

SurfaceYellowDuck opened the pull request:
Add dualport memory for support Tang Primer 20K.

SurfaceYellowDuck committed:
patch: add support dp_memory for tang primer 20k board

RozhkovAleksandr commented:
Хм, это и правда очень полезный код
Рекомендую ознакомиться

achuykov-sc commented:
@SurfaceYellowDuck и @RozhkovAleksandr,

У, да это интересно, вы можете поделиться скриптами сборки FPGA проекта? 

Best regards,
Alexander

SurfaceYellowDuck commented:
@achuykov-sc , Здравствуйте! Со скриптом сборки вы можете ознакомиться в пулл реквесте в репозиторий         
fpga-sdk-prj по [ссылке](https://github.com/syntacore/fpga-sdk-prj/pull/6/commits/5733b1f19e5274b18dfccbf1cfe64b3d86ed0ed0).

SurfaceYellowDuck committed:
fix: add /* synthesis syn_keep = 1 */ directive
add /* synthesis syn_keep = 1 */ to  tapc_trst_n to avoid optimization of scr1_reset_and2_cell

SurfaceYellowDuck committed:
fix: byte access to memory
Now memory support byte writing

SurfaceYellowDuck committed:
fix: endianess in dp mem
